oat-sa/extension-tao-devtools
# Feature/tao-7971 globe icon

Related to: https://oat-sa.atlassian.net/browse/TAO-7971

We need a globe icon which looks like this:

![Screenshot 2019-04-04 at 18 40 04](https://user-images.githubusercontent.com/4357095/55572996-26df4780-5709-11e9-9c4e-c9dc5f950125.png)

How to test
On local machine:

Open Tao
login as admin
navigate to tools > TAO Icon Font, you should see the icon here along with the rest.
